tern/analyze/common.py

```diff
@@ -9,10 +9,11 @@
 
 import logging
 import os
+import re
 import subprocess  # nosec
 
-from tern.classes.package import Package
 from tern.classes.file_data import FileData
+from tern.classes.package import Package
```

Comment:
Small nit: rearrange the placement of this line so it doesn't show up as a change.

Reply:
Done